tests/nnstreamer_query/runTest.sh

```diff
@@ -47,6 +54,47 @@ callCompareTest raw3_0.log result3_0.log 3-3 "Compare 3-3" 1 0
 callCompareTest raw3_1.log result3_1.log 3-4 "Compare 3-4" 1 0
 callCompareTest raw3_2.log result3_2.log 3-5 "Compare 3-5" 1 0
 
+check_mosquitto() {
+    pid=`ps aux | grep mosquitto | grep -v grep | awk '{print $2 }'`
+
+    if [ $pid > 0 ]; then
+        echo "mosquitto broker is running"
+        return 1
+    fi
+    echo "mosquitto broker is not running"
+    return 0
+}
+
+# Check whether mqtt broker is running or not
+check_mosquitto
+ret=$?
+if [  $ret -ne 1 ]; then
+    echo "Warning! mosquitto is not running so skip MQTT-hybrid test."
+    rm *.log
+    report
+    exit
+fi
```

Comment:
```suggestion

    if [ $pid > 0 ]; then
        echo "mosquitto broker is running"
        return
    fi
    echo "Warning! mosquitto is not running so skip MQTT-hybrid test."
    rm *.log
    report
    exit
}
```
This is just a suggestion. BTW, I think it is possible to make these lines (from line 57 to 76) without ```check_mosquitto()```.

Reply:
Thanks for the suggestion. updated.